src/DynamoCore/Graph/Workspaces/SerializationConverters.cs

```diff
@@ -97,7 +97,15 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
             NodeModel node = null;
 
             var obj = JObject.Load(reader);
-            var type = Type.GetType(obj["$type"].Value<string>());
+            Type type = null;
+            try
+            {
+               type = Type.GetType(obj["$type"].Value<string>());
+            }
+            catch(Exception e)
+            {
+                nodeFactory?.AsLogger().Log(e);
+            }
```

Comment:
@mjkkirschner do we want to continue to look into the preloaded nodes for the types if the exception happens?